Projects/count-down/README.md

```diff
@@ -0,0 +1,4 @@
+# COUNT-DOWN TIMER
```

Comment:
Why is the count-down timer mentioned when the specified Issue and PR are for Tic-Tac-Toe?